packages/block-library/src/quote/v2/save.js

```diff
@@ -1,11 +1,20 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+
 /**
  * WordPress dependencies
  */
 import { InnerBlocks, RichText, useBlockProps } from '@wordpress/block-editor';
 
 export default function save( { attributes } ) {
-	const { citation } = attributes;
-	const blockProps = useBlockProps.save();
+	const { align, citation } = attributes;
```

Comment:
It's so bad that this attribute is called "align" instead of "textAlign" as "align" is also a reserved attribute for the block alignments hook. But I guess that's probably not limited to v2 of quote and may exist in other blocks too.